lib/http.js

```diff
@@ -1344,6 +1344,22 @@ Agent.prototype.removeSocket = function(s, name, host, port, localAddress) {
 var globalAgent = new Agent();
 exports.globalAgent = globalAgent;
 
+function check_is_token(val) {
```

Comment:
`check_is_token` sounds a bit generic, even though this is in `lib/http.js`. Maybe `check_is_http_token` is more explicit?